services/blockchain-connector/shared/sdk/dynamicReward.js

```diff
@@ -13,18 +13,22 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const { Exceptions: { TimeoutException } } = require('lisk-service-framework');
+const {
+	Logger,
+	Exceptions: { TimeoutException },
+} = require('lisk-service-framework');
 
 const { timeoutMessage, invokeEndpoint } = require('./client');
 
+const logger = Logger();
 let rewardTokenID;
 
 const getRewardTokenID = async () => {
 	if (!rewardTokenID) {
 		try {
 			// TODO: Update endpoint once exposed by SDK
 			// Ref: https://github.com/LiskHQ/lisk-sdk/issues/7834
-			const response = await invokeEndpoint('reward_getRewardTokenID');
+			const response = await invokeEndpoint('dynamicReward_getRewardTokenID');
```

Comment:
When invoking a `dynamicReward_*****` method fails, we should also try invoking the corresponding `reward_*****` method before throwing the error (implicitly). This is required since some applications might opt to register the `reward` module instead of the `dynamicReward` module in their applications.

Reply:
Applied.